fireflies/data/fireflies/functions/init.mcfunction

```diff
@@ -0,0 +1,21 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing FireFlies..."}]
+execute unless entity @p run say GM4: Installing FireFlies...

```

Comment:
Needs to be updated to the latest format - look at other data packs for a reference